composer.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "sdokus/demo-ajax-inspector",
+    "description": "A Demo WordPress plugin by Sami Dokus",
+    "type": "wordpress-plugin",
+    "license": "GPL-2.0",
+    "autoload": {
+        "psr-4": {
+            "Ajax_Inspector\\": "src/Ajax_Inspector"
```

Comment:
Lets make sure we have your Namespace reflected here. `SDokus` or `Sdokus`, avoid it all being uppercase.